pandas/_libs/groupby.pyx

```diff
@@ -414,8 +414,8 @@ def group_any_all(uint8_t[::1] out,
         String object dictating whether to use any or all truth testing
     skipna : bool
         Flag to ignore nan values during truth testing
-    use_kleene_logic : bool, default False
-        Whether or not to compute the result using Kleene logic
+    masked : bool
```

Comment:
I'd suggest calling this `is_nullable` instead, with the docstring explaining that we use kleene logic on nullable dtypes.

Reply:
done